starboard/shared/posix/socket_send_to.cc

```diff
@@ -57,6 +57,7 @@ int SbSocketSendTo(SbSocket socket,
     socket->error = sbposix::TranslateSocketErrno(errno);
     return -1;
   } else if (socket->protocol == kSbSocketProtocolUdp) {
+#if !SB_HAS_QUIRK(SENDING_CONNECTED_UDP_SOCKETS)
```

Comment:
nit: I think it's more readable if you remove the not, and swap the blocks (`#if SB_HAS_QUIRK()`).